ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -62,6 +62,15 @@ const (
 	InteractiveMode JobSubmissionMode = "InteractiveMode" // Don't submit job in KubeRay. Instead, wait for user to submit job and provide the job submission ID.
 )
 
+type DeletionPolicy string
+
+const (
+	DeleteClusterDeletionPolicy DeletionPolicy = "DeleteCluster"
```

Comment:
Would you mind adding some comments to explain these policies?

Reply:
done